source/robot_model/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+SOURCE_PATH=${PWD}
+
+apt-get update && apt-get install -y \
+    libeigen3-dev \
+    zlib1g-dev \
+    libssl-dev \
+    libboost-all-dev \
+    libboost-dev \
+    libboost-filesystem-dev \
+    libboost-system-dev \
+    libboost-chrono-dev \
+    libboost-date-time-dev \
+    libboost-thread-dev \
+    libboost-test-dev \
+    libbz2-dev \
+    libncurses-dev \
+    pax \
+    tar \
+    liburdfdom-dev \
+    libassimp-dev \
+    assimp-utils \
+    doxygen \
+    texlive-latex-extra
+
+# install pinocchio
+cd ${SOURCE_PATH}/lib && git clone git://git.openrobots.org/robots/robotpkg
```

Comment:
Unfortunately this line fails for me in the docker build with `can't cd to /usr/lib/robot_model/lib`. I assumed it was permissions related, but then I realised the `lib` may never have been created right? When I add `mkdir ${SOURCE_PATH}/lib` to the line above it works

Reply:
oh you beat me to it with your latest commit, I guess I was too slow and now my comment looks outdated

Reply:
thats funny, we just spent 30mins on that haha